modules/ROOT/pages/update-acm.adoc

```diff
@@ -228,6 +229,32 @@ If you are upgrading from a package version before 1.79, update the permissions
 .. Ensure *Read* is enabled, and in *Field Permissions* ensure all fields have *Read Access* enabled.
 . Check that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
 
+Return to the main list of upgrade instructions to continue.
+
+[[post-1801]]
+== Post-Install Steps When Upgrading from a Version before 1.80.1
+
+If you are upgrading from a package version before 1.80.1, update the permissions of the `Member` and `Guest` profiles on the *AnypointGrantTypes* object, and ensure both `Member` and `Guest` profiles have field-level `Read` access to the `GrantTypes` field in the *AnypointApplications* object:
```

Comment:
Please be sure that you are formatting to reflect a UI element (label or syntax value) and not formatting regular names of things like objects, profiles, access levels, and so on.
GLOBAL